_includes/getting-started.html

```diff
@@ -124,7 +124,7 @@ <h3>Brew User:</h3>
       </div>
 
       <!-- Docker User -->
-      <div id="docker_code" class="toggle hideDiv">
+      <div id="docker_code" class="toggle hideDiv" style="background-color: rgb(84, 89, 91);">
```

Comment:
Don't use hardcoded values for styling. Try to use variables and avoid using inline-styling.